ssas/monitoring/monitoring.go

```diff
@@ -1,11 +1,12 @@
 package monitoring
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 	"os"
 
-	"github.com/CMSgov/bcda-ssas-app/ssas"
+	"github.com/CMSgov/bcda-app/log"
```

Comment:
This is an import for the bcda-app; should this be referencing another package in the SSAS-app [where the context logger](https://github.com/CMSgov/bcda-ssas-app/pull/151/files#diff-330686dc447b6523e8cbb02ae01e63ed53f3b214274ecc0da55c978965dda5ddR79) lives?